modules/layers/src/base-tile-layer/utils/tile.js

```diff
@@ -5,10 +5,13 @@ export default class Tile {
     this.z = z;
     this.bbox = tileToBoundingBox(this.x, this.y, this.z);
     this.isVisible = true;
-    this.getTileData = getTileData;
```

Comment:
This class is called `Tile3DHeader` in 3d tiles, as it is not the tile contents but the metadata. Could it make sense to align?

Reply:
This class contains both the metadata and the content. Where is the content stored in 3d tiles?